ena-submission/scripts/call_loculus.py

```diff
@@ -77,42 +74,39 @@ def make_request(
     json_body: dict[str, Any] | None = None,
     data: str | None = None,
 ) -> requests.Response:
-    """
-    Generic request function to handle repetitive tasks like fetching JWT and setting headers.
-    """
+    """Generic request function to handle repetitive tasks like fetching JWT and setting headers."""
     jwt = get_jwt(config)
-    if headers:
-        headers["Authorization"] = f"Bearer {jwt}"
-    else:
-        headers = {"Authorization": f"Bearer {jwt}"}
+    headers = headers or {}
+    headers["Authorization"] = f"Bearer {jwt}"
 
     match method:
         case HTTPMethod.GET:
-            response = requests.get(url, headers=headers, params=params)
+            response = requests.get(url, headers=headers, params=params, timeout=60)
         case HTTPMethod.POST:
             if files:
                 headers.pop("Content-Type")  # Remove content-type for multipart/form-data
-                response = requests.post(url, headers=headers, files=files, data=params)
+                response = requests.post(url, headers=headers, files=files, data=params, timeout=60)
             else:
                 response = requests.post(
-                    url, headers=headers, json=json_body, params=params, data=data
+                    url, headers=headers, json=json_body, params=params, data=data, timeout=60
                 )
         case _:
-            raise ValueError(f"Unsupported HTTP method: {method}")
+            msg = f"Unsupported HTTP method: {method}"
+            raise ValueError(msg)
+
+    if response.status_code != HTTPStatus.OK:
```

Comment:
Is it intentional to not use `response.ok`?

Reply:
Yes we can just use that - I changed now to be more consistent